scripts/build.js

```diff
@@ -8,12 +8,9 @@ const bytes = require("bytes");
 
 const minify = true;
 const v8cache = true;
-const cache = join(__dirname, "..", ".cache");
+const cache = process.argv[2] === "--no-cache" ? false : join(__dirname, "..", ".cache");
```

Comment:
In the cache case, the ncc default is always to cache so we shouldn't need to provide a custom path either.